vanilla-framework/vanilla-docs-theme
# Add new ./run script for building

Inline with the latest methods for local development recently added to
www.ubuntu.com project, I am adding a similar ./run script to this project.

You can now run any of:

- `./run build`
- `./run watch`
- `./run test`
- `./run clean`

Each task will be run within Docker containers, so you don't need yarn or
npm locally, and you don't need to worry about whether you're using the
correct version.

NPM dependencies will also be automatically managed so you don't need to
ever run `npm install` or `yarn install`.

QA
---

Start with `./run --help`. Then try running the available commands. Check they work as expected.

Also, read [the new README](https://github.com/nottrobin/vanilla-docs-theme/blob/run-script/README.md).
